283.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {number[]} nums
+ * @return {void} Do not return anything, modify nums in-place instead.
+ */
+var moveZeroes = function (nums) {
+  let j = 0;
+  let i = 1;
+  while (i < nums.length) {
```

Comment:
```suggestion  

  while (i < nums.length) {
```
It's good to add some space between groups of code for readability, which are the declarations of variables and the others here.